map_gui/src/render/map.rs

```diff
@@ -466,4 +471,16 @@ impl DrawMap {
         let item_id = self.lane_ids.remove(&l).unwrap();
         self.quadtree.remove(item_id).unwrap();
```

Comment:
Question re: `Lane`s vs `Road`s.

This is maybe more related to the previous PR, but don't we also need to update the quadtree's bbox for the lane's Road whenever we add or remove a lane?

Orrrr do we not even use the `Road` in the quadtree and only use `Lane`s?

Reply:
Good catch -- fixed. Already making use of that `quadtree_ids` refactor. :)

To test this, I made a very wide road and used the unzoomed road mouseover. I had to make it reaaaaaally wide to make sure the previous quadtree bin was getting exceeded:
![Screenshot from 2021-04-10 11-48-03](https://user-images.githubusercontent.com/1664407/114281485-7b51e780-99f3-11eb-88f7-ddb940a171a8.png)
I think some graves in this cemetery may have been displaced by this change. :(